tests/suites/cloud/tests/multicontainer/index.js

```diff
@@ -192,7 +192,7 @@ module.exports = {
             this.balena.uuid
           )
           return deviceHash === commit
-        })
+        }, false, 60, 5 * 1000);
```

Comment:
This also halves the total time, 5min vs 10min. Was that intentional? Haven't [we seen devices fail](https://github.com/balena-os/balena-beaglebone/issues/383) to reach the target state in the existing time window?

Reply:
That's a fair point, on a slow connection/device, this image could take more time to download/apply.